force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -94,7 +129,14 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the input is "Empty", "Null", or "Contains Text".
      */
     public static String checkString(String a) {
-        return null; // Replace null with the variable you used to store the result
+        if (a == null) {
+            return 'Null';
+        } else if (a.isEmpty()) {
```

Comment:
You can not call isEmpty this way. It is a static method on the String class. 

String.isEmpty(a)